.circleci/check-testnet-deployment.js

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env node
```

Comment:
We don't need this code. It can be solved by importing nebula and using metrics-related methods from there.

Reply:
After going through your metrics related code in Nebula  - I am not quite sure it's the same.
The method `waitForSync()` waits for 1 endpoint to sync. My code here is not testing for this.
Syncing can happen also with mismatched versions and that's not what I'm aiming for.
I'm not moving forward until I see the network is completely updated to the new version as a deployment should be.

I agree that we can replace my code for `eventuallyClosingBlocks()` and use your `waitForSync()`
What do you think ?